WooCommerce/Classes/ViewRelated/Dashboard/Factories/StatsDataTextFormatter.swift

```diff
@@ -32,6 +32,27 @@ struct StatsDataTextFormatter {
         return createDeltaPercentage(from: previousRevenue, to: currentRevenue)
     }
 
+    /// Creates the text to display for the net revenue.
+    ///
+    static func createNetRevenueText(orderStats: OrderStatsV4?, currencyFormatter: CurrencyFormatter?, currencyCode: String) -> String {
+        guard let revenue = orderStats?.totals.netRevenue else {
+            return Constants.placeholderText
+        }
+
+        // If revenue is an integer, no decimal points are shown.
+        let numberOfDecimals: Int? = revenue.isInteger ? 0 : nil
+        let currencyFormatter = currencyFormatter ?? CurrencyFormatter(currencySettings: ServiceLocator.currencySettings)
```

Comment:
nit: I feel this is more readable when we declare default value near the non-optional func parameter.
Now passing `nil` means using default value, but it is not obvious from outside of the function.